zone/matches.php

```diff
@@ -727,6 +727,67 @@ public function get_team_civs(int $team1_id, int $team2_id, int $map_id): array
         return $teams;
     }
 
+    private static function get_free_pick_civ_id()
+    {
+        return config::get(config::free_pick_civ_id);
+    }
+
+    public function get_banned_civs(int $match_id, int $map_id): array
+    {
+        $freePickCivId = strval($this->get_free_pick_civ_id());
```

Comment:
Please don't use camel case for PHP variables

Reply:
bäääääääh